components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -341,6 +348,7 @@ function ImageBuildView(props: { workspaceId: string, phase?: StartPhase, error?
     <Suspense fallback={<div />}>
       <WorkspaceLogs logsEmitter={logsEmitter} errorMessage={props.error?.message} />
     </Suspense>
+    <button className="mt-6 secondary" onClick={props.onStartWithDefaultImage}>Start with Default Image</button>
```

Comment:
**nitpick**: What do you think of rephrasing this to align with the voice and tone of the rest of the product? This is minor, feel free to leave it as is. 🏓 


```suggestion
    <button className="mt-6 secondary" onClick={props.onStartWithDefaultImage}>Continue with Default Image</button>
```